analytics_data_api/v0/serializers.py

```diff
@@ -332,6 +332,14 @@ class LearnerSerializer(serializers.Serializer, DefaultIfNoneMixin):
     enrollment_date = serializers.DateField(source='enrollment_date', format=settings.DATE_FORMAT)
     cohort = serializers.CharField(source='cohort')
 
+    def transform_segments(self, _obj, value):
+        # returns null instead of empty strings
+        return value or None
```

Comment:
I actually thought that we were getting the desired behavior with segments, which should be a list of strings. If the user belongs to no segments, I'd expect an empty list, not `null`. What do you think?